src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -0,0 +1,188 @@
+@import (reference) "../less/utilities";
+
+// @import (reference) "~clever-components/dist/less/index";
+
+@ICON_SMALL_SIZE: @size_2xl;
+@ICON_REGULAR_SIZE: @size_2xl;
+@HOVER_BACKGROUND_COLOR: #f9faff;
+@ERROR_RED: #df3537;
+@ERROR_RED_TEXT: #b30c10;
+@WHITE: #fff;
```

Comment:
This one's `@neutral_white` from /src/less/colors.less.
https://github.com/Clever/components/blob/master/src/less/colors.less#L23